src/components/Home.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { Card } from 'semantic-ui-react';
+import { getOneCoin } from '../redux/coinsAPI';
+import '../styles/home.css';
+
+const Home = () => {
+  const data = useSelector((state) => state.coins);
+  const dispatch = useDispatch();
+  const oneCoin = (e) => {
+    dispatch(getOneCoin(data, e.target.id));
+  };
+
+  return (
+    <div
+      className="grid"
+      role="button"
+      tabIndex={0}
+      key="main"
+      onClick={oneCoin}
+      onKeyDown={oneCoin}
+    >
+      {
+          data.map((item) => (
+            <NavLink
+              style={{ textDecoration: 'none', color: 'black' }}
+              className="items"
+              key={item.name}
+              name={item.name}
+              id={item.id}
+              to="/Details"
+            >
+              <div key={item.id} id={item.id}>
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.

Reply:
Thank you for this😁😁! I totally forgot that we have to use a filter to let the user choose faster the item that he want to see!